src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days > 2 && days < 7) {
```

Comment:
you can place this condition after `6 days` one to omit additional check
```suggestion
  if (days > 2) {
```